apps/condo/domains/ticket/access/TicketReportService.js

```diff
@@ -0,0 +1,13 @@
+async function canReadTicketReportWidgetData ({ authentication: { item: user } }) {
+    if (!user) return false
+
+    if (user.isAdmin) {
+        return {}
```

Comment:
We can also return true if we want to give permission to get all data

Reply:
Do the support need to access these numbers? This data might get handy in some cases.